sample-apps/react/react-video-demo/src/components/LobbyPanel/LobbyPanel.tsx

```diff
@@ -115,8 +119,8 @@ export const LobbyPanel: FC<Props> = ({
         </div>
 
         <VideoPreview
-          DisabledVideoPreview={() => <DisabledVideoPreview name={user.name} />}
-          NoCameraPreview={() => <DisabledVideoPreview name={user.name} />}
+          DisabledVideoPreview={() => permissionsErrorComponent}
```

Comment:
I believe, we do not need to keep the component in a state, because the update would be triggered by the changed value of prop permissionEnabled. That means that we could do here:

```suggestion
          DisabledVideoPreview={(permissionsEnabled ? EnableBrowserSettings :  DisabledVideoPreview }
```

The `DisabledVideoPreview` can probably take the name from the `useUserContext();` and avoid prop drilling.